### ISSUE TITLE
[BITV] 9.1.3.2/3.1 - Modal dialog "Choose target folder" - When the dialog is open, it is still possible to navigate through the elements in the background using a screen reader, which should be prevented. (2)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36925]
- [x] https://github.com/nextcloud/server/pull/37780

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-1782286125-v3-n1

</details>

- [x] 9\.1.3.2/10.1 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-1782286125-v10-n1
